### PR TITLE
Editting doc for clarity.

### PR DESCRIPTION
**Pull request by alchemi5t**
https://discourse.mozilla.org/t/tutorial-how-i-trained-a-specific-french-model-to-control-my-robot/22830/8

There are quite a few people who couldn't figure out where to find generate_trie. just adding a small reference to hopefully give clarity.

### CODE REVIEW

**Comment by kdavis-mozilla** [comment]
Could you squash your commits?

File: README.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ also, if you need some binaries different than current master, like `v0.2.0-alph
 python3 util/taskcluster.py --branch "v0.2.0-alpha.6" --target "."
 ```
 
-The script `taskcluster.py` will download `native_client.tar.xz` (which includes the `deepspeech` binary and associated libraries) and extract it into the current folder. Also, `taskcluster.py` will download binaries for Linux/x86_64 by default, but you can override that behavior with the `--arch` parameter. See the help info with `python util/taskcluster.py -h` for more details. Specific branches of DeepSpeech or TensorFlow can be specified as well.
+The script `taskcluster.py` will download `native_client.tar.xz` (which includes the `deepspeech` binary, generate_trie utility and associated libraries) and extract it into the current folder. Also, `taskcluster.py` will download binaries for Linux/x86_64 by default, but you can override that behavior with the `--arch` parameter. See the help info with `python util/taskcluster.py -h` for more details. Specific branches of DeepSpeech or TensorFlow can be specified as well.
```
Comment (lissyx) [inline README.md]: Can you use backquotes here around `generate_trie` ?

File: data/README.md [hunk 2]
```diff
@@ -4,6 +4,8 @@ This directory contains language-specific data files. Most importantly, you will
 
 1. A list of unique characters for the target language (e.g. English) in `data/alphabet.txt`
 2. A binary n-gram language model compiled by `kenlm` in `data/lm/lm.binary`
-3. A trie model compiled by `generate_trie.cpp` in `data/lm/trie`
+3. A trie model compiled by [generate_trie](https://github.com/mozilla/DeepSpeech#using-the-command-line-client) (in native_client) in `data/lm/trie`
 
-For more information on how to create these resources, see `data/lm/README.md`
+
+You can find previous versions for generating trie (and pre-built binaries) from [here](https://github.com/mozilla/DeepSpeech/releases). 
+ If you want to build these resources from scratch, see `data/lm/README.md`
```
Comment (lissyx) [inline data/README.md]: There's an extra space at the start of the line.

File: data/README.md [hunk 3]
```diff
@@ -4,6 +4,8 @@ This directory contains language-specific data files. Most importantly, you will
 
 1. A list of unique characters for the target language (e.g. English) in `data/alphabet.txt`
 2. A binary n-gram language model compiled by `kenlm` in `data/lm/lm.binary`
-3. A trie model compiled by `generate_trie.cpp` in `data/lm/trie`
+3. A trie model compiled by [generate_trie](https://github.com/mozilla/DeepSpeech#using-the-command-line-client) (in native_client) in `data/lm/trie`
 
-For more information on how to create these resources, see `data/lm/README.md`
+
+You can find previous versions for generating trie (and pre-built binaries) from [here](https://github.com/mozilla/DeepSpeech/releases).
```
Comment (lissyx) [inline data/README.md]: I'm not sure about that, both the wording and adding a link to releases, especially inside `data/README.md`

Comment (alchemi5t) [reply data/README.md]: Got it! fixing.

**Comment by alchemi5t** [comment]
@kdavis-mozilla squashed.
@lissyx I think I've done everything you requested. Please let me know there's more that needs to be done.

**Comment by lissyx** [comment]
@alchemi5t why closing?

**Comment by alchemi5t** [comment]
I think i messed up the squashing bit. i'll send a new PR.  i think i squashed older commits(not mine) to it.

**Pull request closed without merging**